eng/PackageReference/ProjectReunion.InteractiveExperiences/Microsoft.ProjectReunion.InteractiveExperiences.TransportPackage.PackageReference.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- This csproj is to facilitate restoring dependencies from Maestro. These versions are pulled from eng/version.details.xml -->
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I thought WinAppSDK 1.6 added net8.0 support.  I'm surprised net6 is still being used in the main branch.